api/v1beta1/verticadb_types.go

```diff
@@ -619,6 +619,18 @@ type Subcluster struct {
 	// specify. If not set, the external IP list is left empty in the service object.
 	// More info: https://kubernetes.io/docs/concepts/services-networking/service/#external-ips
 	ExternalIPs []string `json:"externalIPs,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// Specify IP address of LoadBalancer service for this subcluster.
+	// This field is ignored when serviceType != "LoadBalancer".
+	// More info: https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer
+	LoadBalancerIP string `json:"loadBalancerIP,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// A map of key/value pairs appended to service metadata.annotations.
+	ServiceAnnotations map[string]string `json:"serviceAnnotations,omitempty"`
```

Comment:
You can set annotations at `.spec.annotations` and this will be set for all of the objects created, including service objects.  I guess one advantage to define this here is the annotations will only be set for a particular service object.  If you had multiple subclusters each with their own service objects, are there use cases where you would want different annotations ?  I'm guessing there is, but I wanted to confirm.